src/js/models/Trivia.js

```diff
@@ -0,0 +1,24 @@
+import {createQuestion} from "../view/createQuestion"
+export const triviaStart = document.querySelector('.pokemonTrivia__container');
+export class Trivia {
+    constructor(parent){
+        const pokemonTrivia__title = document.createElement('h1')
+        pokemonTrivia__title.classList.add("pokemonTrivia__title")
+        pokemonTrivia__title.textContent= 'Bienvenido a la trivia Pokemon'
+        const pokemonTrivia__text = document.createElement('p');
+        pokemonTrivia__text.classList.add('pokemonTrivia__text')
+        pokemonTrivia__text.textContent = "Puedes responder correctamente las siguientes preguntas?"
+        const triviaButton = document.createElement('button');
+        triviaButton.classList.add('pokemonTrivia__button');
+        triviaButton.textContent= 'Iniciar Trivia Pokemon'
+        triviaButton.addEventListener('click',()=>{
+            triviaStart.remove();
+            createQuestion('Cual de estos Pokemones es tipo Planta');
```

Comment:
same as text constant suggestion on top, store it under constants file an import it here.

Reply:
done